packages/playground/src/components/execution/stage-execution.tsx

```diff
@@ -68,9 +69,14 @@ const StepNavigation: React.FC<StepNavigationProps> = ({
   )
 }
 
-export const StageExecution: React.FC<StageExecutionProps> = ({ stage, selectedStepIdx, logs }): React.ReactElement => {
+export const StageExecution: React.FC<StageExecutionProps> = ({
+  stage,
+  selectedStepIdx,
+  logs,
+  onEdit
+}): React.ReactElement => {
   if (!stage || !stage?.steps) {
-    return <Text>No steps found</Text>
+    return <></>
```

Comment:
is this suppose to return no text now?

Reply:
Yeah, this was just added temporarily, so just removing it. We don't have the right zero state for this component as of now.